src/index.css

```diff
@@ -21,7 +24,23 @@ code {
   font-weight: 400;
   color:#333;
 }
-.btn-default{
 
 
+.image-hero{
+  transform: rotate(0deg) !;
+top:0;
+left: -500px;
+top:0
+
+
+}
+
+@media screen and  (max-width:768px) {
+  .image-hero{
+    transform: rotate(-90deg);
+   top:200px;
+  left: 0;
+  
+  }
+
```

Comment:
doing it with tailwind would be much better

Reply:
due to difficulties to tweak position absolute, I use pure css.